cmd/src/batch_exec.go

```diff
@@ -30,10 +32,12 @@ const (
 )
 
 type executorModeFlags struct {
-	timeout time.Duration
-	file    string
-	tempDir string
-	repoDir string
+	timeout           time.Duration
+	file              string
+	tempDir           string
+	repoDir           string
+	workspaceFilesDir string
+	api               *api.Flags
```

Comment:
I think this is unused

Reply:
Good catch. This was left over code when I was just trying to get things to work